actb_client/jobs.json

```
@@ -1 +1 @@
-{"spawnrefsmalloffice": [{"simId": "384417fc-fbbb-40e7-89af-0cfcfabbf536", "url": "http://127.0.0.1:80", "placeholder": null}]}
\ No newline at end of file
+{"spawnrefsmalloffice": [], "spawnrefmediumoffice": [], "spawnrefsmalloffice-variant1": []}
```

Just adding in a final line feed at the end of the file
```suggestion
{"spawnrefsmalloffice": [], "spawnrefmediumoffice": [], "spawnrefsmalloffice-variant1": []}

```